app/pubspec.yaml

```diff
@@ -53,6 +53,7 @@ dependencies:
   ulid: '2.0.1'
   tar: '2.0.0'
   api_builder:
+  simple_mustache: ^2.1.0
```

Comment:
we can't pull in arbitrary dependencies.

Reply:
I'm not using this package for the parsing code, don't worry. 

In the previous thread you said not to focus on the UI elements of the parsing yet, so I didn't integrate this into the standard Pub package page, and instead make some quick-and-dirty HTML to show off the results. That's what this package is for and should not be used in the final site.